README.md

```diff
@@ -1,21 +1,29 @@
-# yamda-npm
-
-### Installation
-`npm i yamda`
-
-### Documentation
-[aperture](Docs/aperture.md)  
-[binary](Docs/binary.md)   
-[has](Docs/has.md)      
-[nth](Docs/nth.md)      
-[reduce](Docs/reduce.md)  
-[apply](Docs/apply.md)    
-[divide](Docs/divide.md)   
-[map](Docs/map.md)      
-[pick](Docs/pick.md)     
-[reject](Docs/reject.md)  
-[assoc](Docs/assoc.md)  
-[flatten](Docs/flatten.md)  
-[memoize](Docs/memoize.md)  
-[range](Docs/range.md)    
-[sort](Docs/sort.md)
+Description
```

Comment:
It isn't easily apparent to me what your specs are. Writing them out with markdown checkboxes in a bulleted list will be helpful once project reviews come around next Monday.